pkg/images/enricher/enricher_impl.go

```diff
@@ -575,6 +575,10 @@ func (e *enricherImpl) enrichWithSignature(ctx context.Context, enrichmentContex
 
 	matchingRegistries, err := getMatchingRegistries(registrySet.GetAll(), img)
 	if err != nil {
+		// Do not return an error for internal images when no integration is found.
```

Comment:
Why is this change needed?

Reply:
When looking at the failing test, we expect the enricher to _not_ return an error when no registries are available and `enrichmentContext.Internal` is given. Had to fix this within the `enrichWithSignature`.